openmrs-distro.properties

```diff
@@ -6,9 +6,9 @@ parent.artifactId=${parentArtifact}
 parent.version=${parentVersion}
 parent.type=${parentType}
 
-config.${configArtifact}=${configVersion}
-config.${configArtifact}.groupId=${configGroup}
-config.${configArtifact}.type=${configType}
+content.${contentArtifact}=${contentVersion}
+content.${contentArtifact}.groupId=${contentGroup}
+content.${contentArtifact}.namespace=
```

Comment:
what does the namespace do?  (assumedly a feature we aren't using?)

Reply:
See:  https://openmrs.atlassian.net/browse/SDK-363